<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SilVerPLuM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icons/application-128.png</normaloff>:/resources/icons/application-128.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPlay">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Play</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QStackedWidget" name="playStackedWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="playLauncherPage">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="btnApplicationSettings">
                   <property name="text">
                    <string>Application settings</string>
                   </property>
                   <property name="icon">
                    <iconset theme="configure">
                     <normaloff>.</normaloff>.</iconset>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnShowLog">
                   <property name="text">
                    <string>Profile log</string>
                   </property>
                   <property name="icon">
                    <iconset theme="text">
                     <normaloff>.</normaloff>.</iconset>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cmbProfile">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>578</width>
                    <height>181</height>
                   </rect>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="0" column="0">
                    <widget class="QLabel" name="profileInfo">
                     <property name="maximumSize">
                      <size>
                       <width>800</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="textFormat">
                      <enum>Qt::RichText</enum>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                     <property name="wordWrap">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>64</width>
                        <height>64</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>64</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap resource="resources.qrc">:/resources/icons/application-128.png</pixmap>
                      </property>
                      <property name="scaledContents">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;SilVerPLuM&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Stardew Valley launcher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="btnPlay">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>16</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Play</string>
                   </property>
                   <property name="popupMode">
                    <enum>QToolButton::MenuButtonPopup</enum>
                   </property>
                   <property name="arrowType">
                    <enum>Qt::NoArrow</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="playLogPage">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTreeWidget" name="playLogLog">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="columnCount">
                  <number>6</number>
                 </property>
                 <column>
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Time</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Component</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Subcomponent</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Operation</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Message</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="playLogProgress">
                 <property name="maximum">
                  <number>0</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="textVisible">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>You will find more information in the 'Profile log'</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="playLogClose">
                    <property name="text">
                     <string>Close</string>
                    </property>
                    <property name="icon">
                     <iconset theme="window-close">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConfigure">
       <attribute name="title">
        <string>Configure</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="ProfileManagerWidget" name="configProfileList" native="true"/>
          <widget class="ProfileSettings" name="configProfileConfig" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMod">
       <attribute name="title">
        <string>Modify</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="ProfileManagerWidget" name="modProfileList" native="true"/>
          <widget class="ModManagerWidget" name="modManager" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabManage">
       <attribute name="title">
        <string>Manage</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="ProfileManagerWidget" name="manageProfileList" native="true"/>
          <widget class="SavegameManagerWidget" name="savegameManager" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProfileSettings</class>
   <extends>QWidget</extends>
   <header>profilesettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProfileManagerWidget</class>
   <extends>QWidget</extends>
   <header>profilemanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModManagerWidget</class>
   <extends>QWidget</extends>
   <header>modmanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SavegameManagerWidget</class>
   <extends>QWidget</extends>
   <header>savegamemanagerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
